apps/docusaurus/docs/reference/indexer.md

```diff
@@ -0,0 +1,389 @@
+| property_version_v1        | bigint  | The version of the token's properties under schema version 1. Ex. 1                                                                      |
+| to_address                 | String  | This is an Aptos account address to which the token was sent. Ex. "0x123abc456def7890abcdef1234567890abcdef1234"                         |
+| token_amount               | bigint  | The amount of the token transferred in this activity. Ex. 3                                                                              |
+| token_data_id              | bigint  | Unique identifier for this particular token's data. Ex. 123456789                                                                        |
+| token_standard             | String  | Aptos standard that the collection adheres to. Ex. "v1"                                                                                  |
+| transaction_timestamp      | String  | Timestamp when the transaction occurred. Ex. "2024-04-17T02:14:25.68771"                                                                 |
+| transaction_version        | bigint  | Blockchain version of the transaction. Ex. 10000000                                                                                      |
+| type                       | String  | Type of transfer, e.g., "deposit" or "withdrawal". Ex. "0x3::token::DepositEvent"                                                        |
+
+### `nft_metadata_crawler_parsed_asset_uris`
+
+This table allows you to look up the cdn and uris for NFT images / content.
+
+| Field                           | Type   | Description                                                                                      |
+| ------------------------------- | ------ | ------------------------------------------------------------------------------------------------ |
+| animation_optimizer_retry_count | Int    | Number of retries to optimize animation. Ex. 3                                                   |
+| asset_uri                       | String | URI of the asset. Ex. "https://example.com/nft/123"                                              |
+| cdn_animation_uri               | String | Content Delivery Network URI for animation. Ex. "https://cdn.example.com/animations/123"         |
+| cdn_image_uri                   | String | Content Delivery Network URI for image. Ex. "https://cdn.example.com/images/123"                 |
+| cdn_json_uri                    | String | Content Delivery Network URI for JSON metadata. Ex. "https://cdn.example.com/metadata/123.json"  |
+| image_optimizer_retry_count     | Int    | Number of retries to optimize image. Ex. 3                                                       |
+| json_parser_retry_count         | Int    | Number of retries to parse JSON metadata. Ex. 3                                                  |
```

Comment:
These are internal and we shouldn't even expose them. We don't have to include them.

Reply:
Which fields specifically? (This whole table?)